kmod/src/kernelcompat.h

```diff
@@ -434,4 +434,8 @@ static inline int kc_tcp_sock_set_nodelay(struct socket *sock)
 }
 #endif
 
+#ifndef KC_ITER_IS_IOVEC
```

Comment:
Am I missing something?  This doesn't seem to be used anywhere?  Is this a relic from previous code that's since been removed?  Can we just drop this patch?

Reply:
Ohhhh, it was obsolete since `c296bc1959cacb5ab8faedd14759a0f842fdd6ca` which dropped `scoutfs_data_wait_check_iter`. Yep, we can plain drop this patch!